javascript/ql/test/query-tests/Security/CWE-377/InsecureTemporaryFile.expected

```diff
@@ -0,0 +1,53 @@
+nodes
```

Comment:
Is the fact that you're using a `qlref` file the a reason you're not using consistency checking (`// OK` / `// NOT OK`) based tests?

Reply:
It's just simpler to add a `.qlref` file, that's why.  
No deep reasoning.